salt/etcd-proxy/etcd-proxy.conf.jinja

```diff
@@ -4,19 +4,19 @@
 ETCD_DATA_DIR="/var/lib/etcd/"
 ETCD_NAME="{{ grains['id'] }}"
 
-ETCD_LISTEN_CLIENT_URLS="http://0.0.0.0:2379"
+ETCD_LISTEN_CLIENT_URLS="https://0.0.0.0:2379"
 ETCD_LISTEN_PEER_URLS="https://0.0.0.0:2380"
 ETCD_ADVERTISE_CLIENT_URLS="http://{{ grains['caasp_fqdn'] }}:2379"
 
-ETCD_CA_FILE={{ pillar['paths']['ca_dir'] }}/{{ pillar['paths']['ca_filename'] }}
-ETCD_CERT_FILE=/etc/pki/minion.crt
-ETCD_KEY_FILE=/etc/pki/minion.key
-ETCD_TRUSTED_CA_FILE={{ pillar['paths']['ca_dir'] }}/{{ pillar['paths']['ca_filename'] }}
+ETCD_CA_FILE={{ pillar['ssl']['ca_file'] }}
+ETCD_CERT_FILE={{ pillar['ssl']['crt_file'] }}
+ETCD_KEY_FILE={{ pillar['ssl']['key_file'] }}
+ETCD_TRUSTED_CA_FILE={{ pillar['ssl']['ca_file'] }}
 
-ETCD_PEER_CA_FILE={{ pillar['paths']['ca_dir'] }}/{{ pillar['paths']['ca_filename'] }}
-ETCD_PEER_CERT_FILE=/etc/pki/minion.crt
-ETCD_PEER_KEY_FILE=/etc/pki/minion.key
-ETCD_PEER_TRUSTED_CA_FILE={{ pillar['paths']['ca_dir'] }}/{{ pillar['paths']['ca_filename'] }}
+ETCD_PEER_CA_FILE={{ pillar['ssl']['ca_file'] }}
+ETCD_PEER_CERT_FILE={{ pillar['ssl']['crt_file'] }}
+ETCD_PEER_KEY_FILE={{ pillar['ssl']['key_file'] }}
+ETCD_PEER_TRUSTED_CA_FILE={{ pillar['ssl']['ca_file'] }}
 # ETCD_PEER_CLIENT_CERT_AUTH=on
```

Comment:
And I just checked in the docs. Both ETCD_PEER_CLIENT_CERT_AUTH and ETCD_CLIENT_CERT_AUTH default to false so these will need to be set explicitly.